user guides: adapt and update .po and .md files of verification-allos-advanced.md

erciccione opened the pull request:
#1626 has been merged, so now as for the new structure introduced with #1270, we don't need to update each file manually. po4a will do the annoying work for us.

This PR is the result of running for each language:

```bash
export langw="<LANG>"

po4a-updatepo -f text -m _i18n/en/resources/user-guides/verification-allos-advanced.md -p _i18n/$langw/resources/user-guides/weblate/verification-allos-advanced.po && po4a-translate -L UTF-8 -f text -m _i18n/en/resources/user-guides/verification-allos-advanced.md -p _i18n/$langw/resources/user-guides/weblate/verification-allos-advanced.po -l _i18n/$langw/resources/user-guides/verification-allos-advanced.md --keep 0
```

`po4a-updatepo` is used to update the `.po` files, which will be used by translators. `po4a-translate` converts the `.po` files back to markdown.

This workflow will be more automatized in the future by using `po4a(1)` or a script.

erciccione commented:
Looking up the issue with netlify.

selsta commented on _i18n/en/resources/user-guides/verification-allos-advanced.md:
````diff
@@ -116,58 +140,73 @@ gpg:          There is no indication that the signature belongs to the owner.
-4b 9f 31 68 6e ca ad 97 cd b1 75 e6 57 4b f3 07 f8 d1 c4 10 42 78 25 f4 30 4c 21 da 8a ac 18 64
-CertUtil: -hashfile command completed successfully.
-```
+``` SHA256 hash of file monero-gui-win-x64-v0.12.0.0.zip: 4b 9f 31 68 6e ca
````
Will these line breaks show up on the website?

erciccione replied:
It won't. The structure of the guide should remain the same.

selsta replied:
The netlify preview looks broken due to the line breaks: https://deploy-preview-1639--barolo-time-757cf9.netlify.app/resources/user-guides/verification-allos-advanced.html

erciccione replied:
I see it now. I haven't noticed it using my local markdown previewer. Looking into it. The structure got messed up by the first commit wrapping the .md file.

erciccione replied:
So, for some reason there is no way to avoid wrapping usnig the the `po4a-*` tools. I had to create a configuration file and a workflow for po4a to fix this problem. I have the config file saved locally, i'll tweak it to work for all docs and then i'll PR.

There should be no more problems caused by unexpected wrapping in these files.

selsta commented on _i18n/en/resources/user-guides/verification-allos-advanced.md:
````diff
@@ -116,58 +140,73 @@ gpg:          There is no indication that the signature belongs to the owner.
-```
-wget -O monero-linux-x64-v0.15.0.1.tar.bz2 https://downloads.getmonero.org/cli/linux64
-```
+``` wget -O monero-linux-x64-v0.15.0.1.tar.bz2
````
We also don't want a line break e.g. here.

erciccione commented:
Reminder: this is ready for a final review (see https://github.com/monero-project/monero-site/pull/1639#discussion_r638051291)

selsta commented:
I don't know how to review this. It still looks wrong in the netlify preview.

erciccione commented:
@selsta i was wrong. I haven't pushed the fixes yet. Converting this to draft for now.

erciccione commented:
Turns out i did push the fixes, but that reverted a change to a string that i had moved because the structure wasn't compatible with po4a, but was showing ok on the markdown preview of my IDE. Reverted back and updated the .po and .pot files. Everything should be good now.

selsta reviewed (approved):
Checked a couple files and they seem ok.
